docs/source/interfaces/muon_transform_tab.rst

```diff
@@ -3,42 +3,71 @@
+This tab is designed for the user to transform pre-loaded data into the frequency domain. At present there are two
+methods available, which the user can switch between by using the drop-down menu at the top of the tab.
 
-Default: Fast Fourier Transforms
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Fast Fourier Transforms (FFT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. image::  ../images/frequency_domain_analysis_2_transform.png
-   :align: center
+.. image::  ../images/muon_interface_tab_transform.png
+   :align: right
    :height: 500px
 
 In this mode the interface displays two tables, FFT and Advanced options. The FFT table contains
 all of the information required to produce a forward FFT. The Advanced Options table contains the information
 for adding padding and apodization functions to the data.
 
-If the workspace contains the word ``PhaseQuad`` it will run the :ref:`PhaseQuad <algm-PhaseQuad>` algorithm on the selected run when ``Calculate`` is pressed. If no phase table is available then
-the one will be generated and the result will be stored as ``PhaseTable``.
+**Workspace** This selects the workspace to be transformed. If the workspace contains the word ``PhaseQuad`` it will
+run the :ref:`PhaseQuad <algm-PhaseQuad>` algorithm on the selected run when ``Calculate`` is pressed. If no phase table
+is available then the one will be generated and the result will be stored as ``PhaseTable``.
 
-The ``Calculate`` button will execute the :ref:`Padding and Apodization <algm-PaddingAndApodization>`  and then :ref:`FFT <algm-FFT>` algorithms. The output workspace will be added to
-appropriate grouped workspace.
+**Imaginary Data** If unchecked the FFT will be performed without an imaginary component.
 
+**Imaginary Workspace** The imaginary component for the FFT.
 
-Maximum Entropy Method
-^^^^^^^^^^^^^^^^^^^^^^
+**Auto shift** If this is checked it will automatically calculate and apply phase shift, if this is unchecked it will
+ask you to supply a correction (defaults to 0.0).
 
-The maximum entropy method can be used to calculate the frequency domain spectrum. This option uses the :ref:`MuonMaxent <algm-MuonMaxent>` algorithm to calculate the frequency spectrum. The MaxEnt Property table contains
-the basic quantitites used during the calculation. If the ``Use Phase Table`` checkbox is set to true, then an input phase table will be used for the initial phase values. The phase table can be constructed or another phase table can be 
-selected from the drop down menu. When the ``Calculate MaxEnt`` button is pressed, :ref:`CalMuonDetectorPhases <algm-CalMuonDetectorPhases>` is ran first if the phase table is to be ``Constructed``.
+**Use Raw data** If this is checked it will use the raw data from the instrument with its default bins, if it is
```

Comment:
```suggestion
**Fit To Raw Data** If this is checked it will use the raw data for the FFT.
If it is unchecked it will use the rebinned data as specified on the home tab.
--



```